jenkins/trunk.yml

```diff
@@ -3,6 +3,7 @@
     project-type: matrix
     defaults: global
     description: |
+        Do not run this job manually!
```

Comment:
It's OK to run trunk jobs manually, it's not OK to change params
```suggestion
```

Reply:
Thank you, fixed